projects/systelab-components/src/lib/datepicker/datepicker.component.ts

```diff
@@ -365,9 +366,10 @@ export class Datepicker implements OnInit, AfterViewInit, DoCheck, OnDestroy {
 		};
 
 		this.language.firstDayOfWeek = this.i18nService.getFirstDayOfWeek();
-		this.language.dateFormatValue = this.i18nService.getDateFormatForDatePicker(true);
+
+		this.language.dateFormatValue = this.dateFormat ? this.dateFormat : this.i18nService.getDateFormatForDatePicker(true);
 		if (this.currentCalendar) {
-			this.currentCalendar.dateFormat = this.language.dateFormatValue;
+			this.currentCalendar.dateFormat = !this.dateFormat ? this.language.dateFormatValue : this.dateFormat;
```

Comment:
I am not sure to understand this change... 
In case that this.dateFormat is present in line 370, this.language.dateFormat has same value that this.dateFormat... so this statement is irrelevant. You could just use this.language.dateFormatValue as before.
In case that this.dateFormat is not present in line 370, this.language.dateFormat has the value 'this.i18Service.getDateFormatForDatePicker(true)'... so line 372 will always set the first statement.

In summary... I think this change is not needed but maybe I am missing something... ¿?